src/frontend/src/instance/prometheus.rs

```diff
@@ -75,29 +77,63 @@ async fn to_query_result(table_name: &str, output: Output) -> ServerResult<Query
 }
 
 impl Instance {
+    async fn handle_remote_query(
+        &self,
+        ctx: &QueryContextRef,
+        catalog_name: &str,
+        schema_name: &str,
+        table_name: &str,
+        query: &Query,
+    ) -> Result<Output> {
+        let table = self
+            .catalog_manager
+            .table(catalog_name, schema_name, table_name)
+            .await
+            .context(CatalogSnafu)?
+            .context(TableNotFoundSnafu {
```

Comment:
Avoid needless formatting.
```suggestion
            .with_context(|| TableNotFoundSnafu {
```